src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
The class Lottery should not extend ColorSupplier. Instead, use composition over inheritance where you create an instance of ColorSupplier inside the Lottery class and use it to get a random color.